src/config/test-config.js

```diff
@@ -1,21 +1,19 @@
-const githubOrganisationName = 'hms-dbmi-cellenics';
-
 process.env.NODE_ENV = 'test';
 
 module.exports = {
   port: 3000,
   clusterEnv: 'test',
   awsAccountId: '000000000000',
   awsRegion: 'eu-west-1',
-  corsOriginUrl: ['https://test.url2.com', 'https://test.url1.com', 'https://scp.biomage.net'],
-  emailDomainName: 'https://scp.biomage.net',
-  domainName: 'scp.biomage.net',
+  corsOriginUrl: ['https://test.url2.com', 'https://test.url1.com', 'https://test.com'],
+  emailDomainName: 'https://test.com',
+  domainName: 'test.com',
```

Comment:
I suggest that we make the test configuration domain agnostic -- all the tests still pass with updated snaps